### DISCUSSION TITLE
Issue with shuffleLayerConfigurations

### DISCUSSION THREAD

**Discussion by AC1AN** [nftchef/art-engine#64] [Help + Q&A]
Hi, I noticed that when I turn on theshuffleLayerConfigurations the generator always skip a number, for example if I ask for 5 images I'll have a result like 1 2 4 5 6 , with a gape in it
In this example I asked for 10 and the number 2 is missing I have instead number 11 
![shuffleissue](https://user-images.githubusercontent.com/100198428/155518525-8c4e2ff4-f448-40f6-970f-b20f7256b56c.png)
.

Thank you

**Comment by nftchef** [comment]
Opening Issue #66 to investigate. TY
